apps/framework-cli/src/cli.rs

```diff
@@ -1056,6 +1073,13 @@ pub async fn cli_run() {
     };
 }
 
+pub async fn connect_to_temporal() -> Result<WorkflowServiceClient<Channel>> {
```

Comment:
Should this be elsewhere? Seems like it's only used in `apps/framework-cli/src/infrastructure/orchestration/temporal.rs`

Reply:
You're correct!  I moved connect_to_temporal to the temporal.rs file.